config/v1/types_infrastructure.go

```diff
@@ -1343,8 +1343,9 @@ type VSpherePlatformSpec struct {
 	// ---
 	// + If VCenters is not defined use the existing cloud-config configmap defined
 	// + in openshift-config.
-	// +kubebuilder:validation:MaxItems=1
 	// +kubebuilder:validation:MinItems=0
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate="",maxItems=1
+	// +openshift:validation:FeatureGateAwareMaxItems:featureGate=VSphereMultiVCenters,maxItems=3
```

Comment:
Does this feature gate already exist?

Reply:
We had the variable before where we capped it to 1 max.  We never supported multiple vCenters.  This feature gate is to allow us to provide that function and use the gate in multiple operators where today we are limiting vCenter count to 1.  In fact, multiple operators have hard coded limit set to one.  With this feature gate, we'll be updating those logic to now allow multiple if this new FeatureGate is enabled.

Reply:
The feature gate will need to be initialised in `features/features.go` else the tooling (and operators) won't actually know the feature gate exists

Reply:
@JoelSpeed , yes, I updated that file in this PR already.  Is that the change you are thinking or did I miss something?

Reply:
Apparently I just couldn't find it, ignore me